api/src/main/java/com/codeforcommunity/rest/ApiRouter.java

```diff
@@ -177,10 +184,70 @@ private void handleLogout(RoutingContext ctx) {
     if (!(ctx.session().isEmpty()) && ctx.session().get("auth").equals(1)) {
       ctx.session().destroy();
       ctx.reroute(ctx.request().path());
+    } else {
+      response.putHeader("content-type", "text/html").end("<h1>Not Logged In</h1>");
     }
+  }
+
+  private void checkAuthentication(RoutingContext ctx) {
+    HttpServerResponse response = ctx.response();
+
+    if (ctx.session().isEmpty() || !ctx.session().get("auth").equals(1)) {
+      response.putHeader("location", "/").setStatusCode(403).end();
+      return;
+    }
+  }
+
+  private void handleCreateMeeting(RoutingContext ctx) {
+    checkAuthentication(ctx);
+
+    HttpServerResponse response = ctx.response();
+    HttpServerRequest request = ctx.request();
+    String id = "";
+    String name = "";
+    LocalDateTime date = null;
+    Boolean open = null;
+    // for now, the input date is to the minute
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```

Comment:
I made the decision that the data (likely coming from some form somewhere?) would probably take the meeting time down to the minute at most. Therefore, the formatter will only take dates to the minute.